arangod/Statistics/StatisticsFeature.cpp

```diff
@@ -195,7 +195,7 @@ void StatisticsFeature::start() {
   }
 }
 
-void StatisticsFeature::unprepare() {
+void StatisticsFeature::stop() {
```

Comment:
this change was already submitted in another (by now already merged) PR of yours.